contrib/sync-upstream.sh

```diff
@@ -100,11 +100,20 @@ git checkout master
 git pull
 git checkout -b temp-merge-"$PRNUM"
 
+# Escape single quote
+# ' -> '\''
+quote() {
+    local quoted=${1//\'/\'\\\'\'}
+    printf "%s" "$quoted"
+}
+TITLE=$(quote "$TITLE")
+BODY=$(quote "$BODY")
+
 BASEDIR=$(dirname "$0")
 FNAME="$BASEDIR/gh-pr-create.sh"
 cat <<EOT > "$FNAME"
 #!/bin/sh
```

Comment:
```suggestion
#!/bin/bash
```

`${1// ... }` is a bashism, see https://stackoverflow.com/a/13210909 . (This has a few POSIX solutions but no need to bother, `sync-upstream.sh` is already bash-specific.)

Reply:
`sync-upstream.sh` already uses bash. The suggested line change doesn't affect `${1// ... }`.

Reply:
Oh man, this PR really confuses me for no reason.  You're right of course.

Reply:
It's mostly just my dumb design of writing yet another shell script to execute. I should fix this at some point.